examples/decompose/do_MLICA.py

```diff
@@ -39,76 +41,97 @@
 reject = {'mag': 5e-12}
 refnotch = [50., 100., 150., 200., 250., 300., 350., 400.]
 
-data_path = op.join(get_jumeg_path(), 'data')
-print(data_path)
-
-# example filname
-raw_fname = "/Volumes/megraid21/sripad/cau_fif_data/jumeg_test_data/" \
-            "109925_CAU01A_100715_0842_2_c,rfDC-raw.fif"
-
-# load the model for artifact rejection
-# the details of the model is provided in the x_validation_shuffle_v4_split_23.txt
-model_name = op.join(data_path, "dcnn_model.hdf5")
-
+# number time samples is fixed to 15000
+nsamples_chop = 15000
+ix_t1 = 0                              # time index: here we use the first chop
+ix_t2 = ix_t1 + nsamples_chop
+
+# ----------------------------------------------
+# load DCNN model for artifact rejection
+# the details of the model is provided in:
+#       x_validation_shuffle_v4_split_23.txt
+# model was trained on 4D data from Juelich
+# ----------------------------------------------
+model_path = op.join(get_jumeg_path(), 'data')
+model_name = op.join(model_path, "dcnn_model.hdf5")
 model = load_model(model_name)
 
+# ----------------------------------------------
+# read example data file
+# ----------------------------------------------
+path_data = '/data/megraid22/Common/DeepLearning/cau_data_validation/'
+raw_fname = path_data + '109925_CAU01A_100715_0842_2_c,rfDC,t1,n_bcc,nr-raw.fif'
+raw = mne.io.Raw(raw_fname, preload=True)
+picks = mne.pick_types(raw.info, meg=True, eeg=False, eog=False,
+                       stim=False, exclude='bads')
+
+# ----------------------------------------------
 # noise reducer
+# ----------------------------------------------
 raw_nr = noise_reducer(raw_fname, reflp=5., return_raw=True)
-
 raw_nr = noise_reducer(raw_fname, raw=raw_nr, refhp=0.1, noiseref=['RFG ...'],
                        return_raw=True)
-
-# 50HZ and 60HZ notch filter to remove noise
 raw = noise_reducer(raw_fname, raw=raw_nr, refnotch=refnotch, return_raw=True)
 
-picks = mne.pick_types(raw.info, meg=True, eeg=False, eog=False,
-                       stim=False, exclude='bads')
-
-raw_filtered = raw.copy().filter(0., 45., picks=picks, filter_length='auto',
+# ----------------------------------------------
+# filtering and down sampling
+# ----------------------------------------------
+# filter prior to ICA
+raw_filtered = raw.copy().filter(flow_raw, flow_high, picks=picks, filter_length='auto',
                                  l_trans_bandwidth='auto', h_trans_bandwidth='auto',
-                                 n_jobs=njobs, method='fir', phase='zero',
+                                 n_jobs='cuda', method='fir', phase='zero',
```

Comment:
Also, maybe its best to set n_jobs=njobs everywhere, and change the default to 'cuda' in line 32 above. Easier for testing on cpu only systems as well.